spotbugs/src/main/java/edu/umd/cs/findbugs/sarif/SarifBugReporter.java

```diff
@@ -87,8 +87,10 @@ private void processInvocations(JsonWriter jsonWriter, @NonNull Map<URI, String>
 
         int exitCode = ExitCodes.from(getQueuedErrors().size(), missingClasses.size(), getBugCollection().getCollection().size());
         Invocation invocation = new Invocation(exitCode,
-                getSignalName(exitCode), exitCode == 0,
-                execNotifications, configNotifications);
+                getSignalName(exitCode),
```

Comment:
Since the second parameter of `Invocation`'s constructor changed to `exitCodeDescription` from `exitSignalName`, the name of this function (the definition is at line 146/148 in this file) also could be changed to represent this.

Reply:
Makes sense, the method name is changed in the latest commit.